src/reconciler.js

```diff
@@ -64,7 +65,7 @@ function updateHost (WIP) {
 
 function getParentNode (fiber) {
   if (!fiber.parent) return fiber.node
-  while (fiber.parent.tag === HOOK) return fiber.parent.parent.node
+  if (fiber.parent.tag === HOOK) return fiber.parent.parent.node

```

Comment:
Note that this change is unrelated to the PR - maybe this was a `while` statement in an earlier version, but an `if` statement should be enough, since all it does it `return` unconditionally?

Reply:
Here still need a while loop, because the parent of the component may still be the component

Reply:
But this `while`-statement doesn't loop.

The body of the `while`-statement is an unconditional `return`-statement - so if the condition in the `while`-statement evaluates true, it returns; exactly like an `if`-statement.

Reply:
Oh, you're right. It's my fault. I'll fix it.